### PR TITLE
More unit tests to IPv6 fragmentation reassembly

### PR DESCRIPTION
**Pull request by rveerama1**

### CODE REVIEW

**Review by tbursztyka — CHANGES_REQUESTED** [review]
Remove the DBG/NET_DBG change (or split it in another patch)

**Review by jukkar — APPROVED** [review]
LGTM

**Comment by rveerama1** [comment]
> Remove the DBG/NET_DBG change (or split it in another patch)

Done.

**Comment by galak** [comment]
If you desire this fix in 1.14, please open or associate this PR with an Issue.

**Review by pfalcon — APPROVED** [review]

**Comment by jukkar** [comment]
@rveerama1 please rebase against latest master and re-submit.

**Comment by rveerama1** [comment]
@jukkar done.

File: tests/net/ipv6_fragment/src/main.c [hunk 1]
```diff
@@ -1055,28 +1050,28 @@ static int verify_fragment(struct net_pkt *pkt)
...
 			return -EINVAL;
 		}
 
 		offset = pkt->buffer->data[6 * 8 + 2] * 256U +
 			(pkt->buffer->data[6 * 8 + 3] & 0xfe);
 
 		if (offset != pkt_recv_data_len) {
-			DBG("Invalid offset %d received %d\n",
+			NET_DBG("Invalid offset %d received %d",
 			    offset, pkt_recv_data_len);
```
Comment (tbursztyka) [inline tests/net/ipv6_fragment/src/main.c]: seems to be badly indented here

File: tests/net/ipv6_fragment/src/main.c [hunk 2]
```diff
@@ -1094,14 +1089,14 @@ static int verify_fragment(struct net_pkt *pkt)
...
+			NET_DBG("");
 			net_hexdump("expected 2", ipv6_second_frag, 7 * 8);
 
 			return -EINVAL;
 		}
 
 		if (pkt_data_len != pkt_recv_data_len) {
-			DBG("Invalid amount of data received (%d vs %d)\n",
+			NET_DBG("Invalid amount of data received (%d vs %d)",
 			    pkt_data_len, pkt_recv_data_len);
```
Comment (tbursztyka) [inline tests/net/ipv6_fragment/src/main.c]: same

**Comment by rveerama1** [comment]
@tbursztyka done.

**Comment by rveerama1** [comment]
@tbursztyka Take a look this one also.

**Review by tbursztyka — APPROVED** [review]

**Pull request merged**